.circleci/config.yml

```diff
@@ -20,6 +20,10 @@ jobs:
       JWT_ACCESS_TOKEN_EXPIRATION_TIME: 86400
       JWT_REFRESH_TOKEN_SECRET: template_refresh
       JWT_REFRESH_TOKEN_EXPIRATION_TIME: 31540000
+      CLOUDINARY_CLOUD_NAME: dfxxthku8
+      CLOUDINARY_API_KEY: 225299895453173
+      CLOUDINARY_API_SECRET: _e8xRjNayHjAIf_oHfba8lImWGA
+
```

Comment:
# IMPORTANT!!!
Please add these new environment variables to your `backend/.env` config file. The confluence page for how to run the app has also been updated to adapt these changes: https://princhcanal.atlassian.net/wiki/spaces/HNDJB/pages/100368431/How+to+run+the+app